### PR TITLE
Update zio 1.0.1 fix

### PR DESCRIPTION
**Pull request by lemastero**
PR #222 cannot be automatically merged. This should solve the problems.

### CODE REVIEW

File: interop-cats/shared/src/main/scala/zio/interop/Schedule.scala [hunk 1]
```diff
@@ -159,7 +159,7 @@ final class Schedule[F[+_], -In, +Out] private (private[Schedule] val underlying
   def combineWith[In1 <: In, Out2](
```
Comment (adamgfraser) [inline interop-cats/shared/src/main/scala/zio/interop/Schedule.scala:159]: In ZIO we deprecated `combineWith` and added the `unionWith` and `intersectWith` operators. I would suggest we do the same here to maintain a consistent API.

**Review by adamgfraser — CHANGES_REQUESTED** [review]
Looks great! Just one suggestion on `Schedule`. Thanks for doing this!

File: interop-cats/shared/src/main/scala/zio/interop/Schedule.scala (hunk 1 above)
Comment (lemastero) [reply interop-cats/shared/src/main/scala/zio/interop/Schedule.scala:168]: Thank you for the review. Deprecated `combineWith` in favour of `intersectWith ` and added `unionWith` as well.

**Review by adamgfraser — APPROVED** [review]
Looks great! Thanks again for working on this! 🙏

**Pull request merged**